languages/csharp/exercises/concept/arrays/.docs/after.md

```diff
@@ -0,0 +1,104 @@
+Data structures that can hold zero or more elements are known as _collections_. An **array** is a collection that has a fixed size and whose elements must all be of the same type. Elements can be assigned to an array or retrieved from it using an index. C# arrays are zero-based, meaning that the first element's index is always zero:
+
+```csharp
+// Declare array with explicit size (size is 2)
+int[] twoInts = new int[2];
+
+// Assign first and second element by index
+twoInts[0] = 7;
+twoInts[1] = 8;
+
+// Retrieve the second element by index
+twoInts[1] == 8; // => true
+
+// Check the length of the array
+twoInts.Length == 2; // => true
+```
+
+Arrays can also be defined using a shortcut notation that allows you to both create the array and set its value. As the compiler can now tell how many elements the array will have, the length can be omitted:
+
+```csharp
+// Assign and initialize (size is 3)
+int[] threeIntsV1 = new int[] { 4, 9, 7 };
+
+// Shorter notation (size is 3)
+int[] threeIntsV2 = new[] { 4, 9, 7 };
+
+// Shortest notation (size is 3)
+int[] threeIntsV3 = { 4, 9, 7 };
+```
+
+Arrays can be manipulated by either calling an array's [methods][array-methods] or [properties][array-properties], or by using the methods defined in the [`Array` class][array-class].
+
+An array is also a _collection_, which means that you can iterate over all its values using a [`foreach` loop][foreach-statement]:
+
+```csharp
+char[] vowels = new [] { 'a', 'e', 'i', 'o', 'u' }; // Size is 5
```

Comment:
Why repeat `Size is X` in many places? and in different formats? (up it is `(size is X)`
To me it should be used when it's relevant... and omitted otherwise.

Reply:
Agreed. I've omitted it here and shortened the other example.